blackbox/docs/general/builtins/scalar.txt

```diff
@@ -591,14 +593,14 @@ the other argument must be a column reference.
 
 .. NOTE::
 
-    The algorithm of the calculation which is used when the distance function
-    is used as part of the result column list has a different precision than
-    what is stored inside the index which is utilized if the distance function
-    is part of a WHERE clause.
+   The algorithm of the calculation which is used when the distance
```

Comment:
don't we always use 2 column identation from the <header> ?

Reply:
no. I've been moving indentation of admonition blocks so that it matches the start of the admonition label. I have documented this in our draft documentation standards, but I have not circulated those standards yet

Reply:
ok